lib-ce/emqx_dashboard/test/emqx_dashboard_SUITE.erl

```diff
@@ -84,9 +88,93 @@ t_admins_add_delete(_) ->
     ?assertNotEqual(true, request_dashboard(get, api_path("brokers"),
         auth_header_("username", "pwd"))).
 
+t_admins_persist_default_password(_) ->
+    emqx_dashboard_admin:change_password(<<"admin">>, <<"new_password">>),
+    ct:sleep(100),
+    [#mqtt_admin{password=Password, tags= <<"administrator">>}] = emqx_dashboard_admin:lookup_user(<<"admin">>),
+
+    %% To ensure that state persists even if the process dies
+    application:stop(emqx_dashboard),
+    application:start(emqx_dashboard),
+
+    ct:sleep(100),
+
+    %% It gets restarted by the app automatically
+    [#mqtt_admin{password=PasswordAfterRestart}] = emqx_dashboard_admin:lookup_user(<<"admin">>),
+    ?assertEqual(Password, PasswordAfterRestart),
+    emqx_dashboard_admin:change_password(<<"admin">>, <<"public">>).
```

Comment:
Nit: if this is to avoid inter-test case dependencies, maybe it's safer to reset it in `end_per_testcase`?